python/tests/simulation.py

```diff
@@ -19,6 +18,10 @@ class TestSimulation(unittest.TestCase):
 
     def setUp(self):
         print("Running {}".format(self._testMethodName))
+        self.temp_dir = mp.make_output_directory()
+
+    def tearDown(self):
+        mp.delete_directory(self.temp_dir)
```

Comment:
This creates a temporary method for every test.   Better to do it in `setUpClass` and `tearDownClass`

Reply:
Done.